Dockerfile.tpl

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6-alpine
```

Comment:
Forgot to change this, will do it as soon as I fix the conflicts and add the new FunctionDeclaration roles.